Assets/HoloToolkit-Examples/Input/Scenes/InputManagerTest.unity

```diff
@@ -1415,7 +1415,7 @@ RectTransform:
   m_PrefabInternal: {fileID: 0}
   m_GameObject: {fileID: 432117531}
   m_LocalRotation: {x: -0, y: -0, z: -0, w: 1}
-  m_LocalPosition: {x: 0.049999237, y: 0.47200012, z: 1.1980001}
+  m_LocalPosition: {x: 0, y: 0, z: 1.1980001}
```

Comment:
Should Z = 0 as well?

Reply:
The editor updated this every time I dismissed it. I'll have to look back and verify what Z should be.